src/main/eo/org/eolang/txt/text.eo

```diff
@@ -69,3 +69,7 @@
 
   [i] > at
     ^.s.slice i 1 > @
+
+  # Returns string where all substrings
+  # substr1 changed to substr2
+  [substr1 substr2] > replaced /string
```

Comment:
@EugeneDar I think that the parameters of this object are incomprehensibly readable. We can call them `target` and `replacement`, or something similar. Got the idea?

Reply:
@Graur I agree, I'll fix it.